components/server/src/ome/services/sharing/make_data.py

```diff
@@ -92,9 +92,14 @@ def compile(dir=dat):
             stderr=subprocess.PIPE,
             shell=True)
         version = proc.communicate()[1].strip()
-        pat = "%s/ice*%s.jar" % (rep, version)
-        cp = ":".join(glob.glob(pat))
-        javac_cmd = "javac -target 1.7 -cp "
+        # Current Ice version
+        pat1 = "~/.m2/repository/com/zeroc/*/%s/*.jar" % (version)
```

Comment:
We're using ivy rather than maven here, and it's hardcoded in `etc/ivysettings.xml`.  This script is only ever run by hand when ice is updated, so if a nonstandard location is used, it might be easier to tweak the script when running it.  Ideally this script wouldn't even exist!